style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.6;
+  color: white;
+}
+
+body::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  background: linear-gradient(rgba(0, 0, 0, 0.5), #042123), url(https://cdn.pixabay.com/photo/2016/08/15/10/17/survey-1594962__340.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+h1 {
+  font-weight: 400;
+  line-height: 1.2;
+}
+
+p {
+  font-size: 1.12rem;
+}
+
+h1,
+p {
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
+
+label {
+  display: flex;
+  align-items: center;
+  font-size: 1.12rem;
+  margin-bottom: 0.5rem;
+}
+
+input,
+button,
+select,
+textarea {
+  margin: 0;
+  font-size: inherit;
+  line-height: inherit;
+  font-family: inherit;
+}
+
+button {
+  border: none;
+}
+
+.container {
+  width: 100%;
+  margin: 3.12rem auto 0 auto;
+}
+
+@media (min-width: 768px) {
+  .container {
+    max-width: 720px;
+  }
+}
+
+@media (min-width: 576px) {
+  .container {
+    max-width: 540px;
+  }
+}
+
+.header {
+  text-align: center;
+  padding: 0 0.625rem;
+  margin-bottom: 1.88rem;
+}
+
+.description {
+  font-weight: 200;
+  font-style: italic;
+  text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.5);
+}
+
+.hint {
+  margin-left: 0.25rem;
+  font-size: 0.93rem;
+  color: #e4e4ef;
+}
+
+form {
+  background: #1b1b32;
+  padding: 2.5rem 0.625rem;
+  border-radius: 0.5rem;
+  opacity: 0.8;
+}
+
+.survey-form {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
```

Comment:
- [OPTIONAL] I had the opportunity to go through both your HTML and CSS codes and I have identified that there is nowhere in your HTML that you've used the class name **survey-form**. This change may be optional but it plays a big role in identifying your professionalism, be sure to remove it :wastebasket: and make your code :sparkles: clean.